docs/site/content/docs/edge/node-pool.md

```diff
@@ -0,0 +1,180 @@
+
+## List Node Pools
+
+To inspect the node pools currently available in a cluster, run:
+
+```shell
+tanzu cluster node-pool list CLUSTER-NAME
+```
+
+This returns a lists of all of the node pools in the cluster, and the state of the replicas in each node pool.
+
+## Create a Node Pool
+
+To create a node pool in a cluster:
+
+1. Create a configuration file for the node pool.  
+    See below for example configuration files for each infrastructure provider.
+
+2. Create the node pool defined by the configuration file:
+
+    ```shell
+    tanzu cluster node-pool set CLUSTER-NAME -f <CONFIG-FILE>
+    ```
+
+    Options:
+
+    - `--namespace` specifies the namespace of the cluster. The default value is `default`.
+    - `--machine-deployment-base` specifies the base `MachineDeployment` object from which to create the new node pool.
+      - Set this value as a `MachineDeployment` identifier as listed in the output of `tanzu cluster get` under `Details`.
+      - The default value is the first in the cluster's array of worker node `MachineDeployment` objects, represented internally as `workerMDs[0]`.
+
+### AWS Configuration
+
+In addition to the required `name`, `replicas`, and `labels` properties above, configuration files for node pools on AWS support the following optional properties:
+
+- `az`: Availability Zone
+- `nodeMachineType`: Instance type
+
+   These settings may be omitted, in which case their values inherit from the cluster's first node pool.
+
+Example node pool definition for an AWS cluster:
+
+```yaml
+name: tkg-aws-wc-np-1
+replicas: 2
+az: us-west-2b
+nodeMachineType: t3.large
+labels:
+key1: value1
+key2: value2
+```
+
+### Azure Configuration
+
+In addition to the required `name`, `replicas`, and `labels` properties above, configuration files for node pools on Microsoft Azure support the following optional properties:
+
+- `az`: Availability Zone
+- `nodeMachineType`: Instance type
+
+These settings may be omitted, in which case their values inherit from the cluster's first node pool.
+
+Example node pool definition for Azure cluster:
+
+```yaml
+name: tkg-azure-wc-np-1
+replicas: 2
+az: 2
+nodeMachineType: Standard_D2s_v3
+labels:
+key1: value1
+key2: value2
+```
+
+### vSphere Configuration
+
+In addition to the required `name`, `replicas`, and `labels` properties above, configuration files for node pools on vSphere can include a `vsphere` block, to define optional properties specific to configuring VMs on vSphere.
+
+Example node pool definition for vSphere cluster:
+
+```sh
+name: tkg-wc-oidc-md-1
+replicas: 4
+labels:
+key1: value1
+key2: value2
+vsphere:
+memoryMiB: 8192
+diskGiB: 64
+numCPUs: 4
+datacenter: dc0
+datastore: iscsi-ds-0
+storagePolicyName: name
+folder: vmFolder
+resourcePool: rp-1
+vcIP: 10.0.0.1
+template: templateName
+cloneMode: clone-mode
+network: network-name
```

Comment:
And then I would also assume these were meant to be indented so they are under the `vsphere` section. Otherwise this doesn't match what was stated above and we're missing a lot.